docs/howtoguides/update_motd_messages.md

```diff
@@ -0,0 +1,12 @@
+# How to update MOTD and APT messages
```

Comment:
I think it'd be nice to add a link to this in the table of contents in the readme as well - what do you think @lucasmoura ?